src/main/java/net/dries007/tfc/objects/recipes/ShapelessDamageFoodRecipe.java

```diff
@@ -56,6 +60,20 @@ public ItemStack getCraftingResult(InventoryCrafting inv)
         return foodStack != null ? CapabilityFood.updateFoodFromPrevious(foodStack, out) : ItemStack.EMPTY;
     }
 
+    @Override
+    @Nonnull
+    public NonNullList<ItemStack> getRemainingItems(final InventoryCrafting inventoryCrafting)
+    {
+        // Give straw to player as well.
+        EntityPlayer player = ForgeHooks.getCraftingPlayer();
+        if (!player.world.isRemote)
```

Comment:
Like you said, missing a check here for ore dictionary: `OreDictionaryHelper.doesStackMatchOre(stack, "grain")`

Reply:
Ooh, I like the oredict helper! Done.